core/src/ldml/ldml_processor.hpp

```diff
@@ -0,0 +1,81 @@
+/*
+  Copyright:    © 2018 SIL International.
+  Description:  Internal keyboard class and adaptor class for the API.
+  Create Date:  2 Oct 2018
+  Authors:      Tim Eves (TSE)
+  History:      2 Oct 2018 - TSE - Refactored out of km_kbp_keyboard_api.cpp
+*/
+
+#pragma once
+
+#include <string>
+#include <vector>
+#include <unordered_map>
+#include <keyman/keyboardprocessor.h>
+
+#include "processor.hpp"
+#include "option.hpp"
+
+namespace km {
+namespace kbp {
+
+#define KM_KBP_LDML_VERSION u"1.0"
```

Comment:
would be better KM_KBP_LMDL_PROCESSOR_VERSION or at least a comment to that effect